### DISCUSSION TITLE
https://kdheepak.com/blog/visualizing-roads-in-the-cities-of-canada/

### DISCUSSION THREAD

**Discussion by giscus** [kdheepak/kdheepak.github.io#16] [Comments]
# https://kdheepak.com/blog/visualizing-roads-in-the-cities-of-canada/

https://kdheepak.com/blog/visualizing-roads-in-the-cities-of-canada/

**Comment by prncevince** [comment]
This is ART; very cool. 🇨🇦 🗺️
